moshi/src/test/java/org/jdbi/v3/moshi/TestMoshiPlugin.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.moshi;
+
+import java.io.IOException;
+
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.JsonReader;
+import com.squareup.moshi.JsonWriter;
+import com.squareup.moshi.Moshi;
+import org.jdbi.v3.core.qualifier.QualifiedType;
+import org.jdbi.v3.json.AbstractJsonMapperTest;
+import org.jdbi.v3.json.Json;
+import org.jdbi.v3.postgres.PostgresDbRule;
+import org.jdbi.v3.testing.JdbiRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestMoshiPlugin extends AbstractJsonMapperTest {
+    @Rule
+    public JdbiRule db = PostgresDbRule.rule();
+
+    @Before
+    public void before() {
+        jdbi = db.getJdbi().installPlugin(new MoshiPlugin())
+                .configure(MoshiConfig.class, c -> c.setMoshi(
+                        new Moshi.Builder().build()));
```

Comment:
I take it this `configure()` is included just to cover the API?

Reply:
Frankly I'm not sure. I practically copy-pasted this test from gson2 test written by @stevenschlansker. But I think you're right, the rationale was to test .configure().